pkg/infrared/infrared.go

```diff
@@ -13,19 +13,15 @@ import (
 )
 
 type Config struct {
-	ListenerConfigs []ListenerConfig
-	ServerConfigs   []ServerConfig
+	Bind          string
```

Comment:
`Bind` might not be an obvious enough name to indicate it could contain both ip and port? :thinking:

Reply:
Good point. I changed it into `BindAddr`